<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMangaViewTab</class>
 <widget class="QWidget" name="CMangaViewTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="toolbar">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnOpenOrigin">
        <property name="toolTip">
         <string>Open source in browser tab</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open-remote">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="toolTip">
         <string>Save manga to disk</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-save">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNavFirst">
        <property name="toolTip">
         <string>Go first</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="go-first">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNavPrev">
        <property name="toolTip">
         <string>Go previous</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="go-previous">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNavNext">
        <property name="toolTip">
         <string>Go next</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="go-next">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNavLast">
        <property name="toolTip">
         <string>Go last</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="go-last">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRotateCCW">
        <property name="toolTip">
         <string>Rotate left</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="object-rotate-left">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRotateCW">
        <property name="toolTip">
         <string>Rotate right</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="object-rotate-right">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="spinPosition">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPosition">
           <property name="text">
            <string>lblPosition</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Loading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZoomFit">
        <property name="toolTip">
         <string>Zoom fit</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="zoom-fit-best">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZoomWidth">
        <property name="toolTip">
         <string>Zoom width</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="zoom-fit-width">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZoomHeight">
        <property name="toolTip">
         <string>Zoom height</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="zoom-fit-height">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZoomOriginal">
        <property name="toolTip">
         <string>Zoom 100%</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="zoom-original">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZoomDynamic">
        <property name="toolTip">
         <string>Dynamic zoom</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="zoom-draw">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboZoom">
        <property name="currentIndex">
         <number>14</number>
        </property>
        <item>
         <property name="text">
          <string>5%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>45%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>55%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>75%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>85%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>95%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Original</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>110%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>125%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>150%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>175%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>250%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>300%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>750%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000%</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ZScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="ZMangaView" name="mangaView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>977</width>
        <height>380</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameLoading">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelLoadInfo">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="loadingProgress">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbortLoading">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">manga/zscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZMangaView</class>
   <extends>QWidget</extends>
   <header location="global">manga/zmangaview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>btnOpenOrigin</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnNavFirst</tabstop>
  <tabstop>btnNavPrev</tabstop>
  <tabstop>btnNavNext</tabstop>
  <tabstop>btnNavLast</tabstop>
  <tabstop>btnRotateCCW</tabstop>
  <tabstop>btnRotateCW</tabstop>
  <tabstop>spinPosition</tabstop>
  <tabstop>btnZoomFit</tabstop>
  <tabstop>btnZoomWidth</tabstop>
  <tabstop>btnZoomHeight</tabstop>
  <tabstop>btnZoomOriginal</tabstop>
  <tabstop>btnZoomDynamic</tabstop>
  <tabstop>comboZoom</tabstop>
  <tabstop>btnAbortLoading</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
